[2025-01-16T03:56:38.754+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: finflow_source_data_pipeline.load_to_gcs_bq.load_account_to_temp manual__2025-01-16T03:55:30.437726+00:00 [queued]>
[2025-01-16T03:56:38.872+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: finflow_source_data_pipeline.load_to_gcs_bq.load_account_to_temp manual__2025-01-16T03:55:30.437726+00:00 [queued]>
[2025-01-16T03:56:38.894+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-01-16T03:56:39.103+0000] {taskinstance.py:1382} INFO - Executing <Task(GCSToBigQueryOperator): load_to_gcs_bq.load_account_to_temp> on 2025-01-16 03:55:30.437726+00:00
[2025-01-16T03:56:39.171+0000] {standard_task_runner.py:57} INFO - Started process 1976 to run task
[2025-01-16T03:56:39.211+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'finflow_source_data_pipeline', 'load_to_gcs_bq.load_account_to_temp', 'manual__2025-01-16T03:55:30.437726+00:00', '--job-id', '1907', '--raw', '--subdir', 'DAGS_FOLDER/finflow.py', '--cfg-path', '/tmp/tmp758rpvzt']
[2025-01-16T03:56:39.265+0000] {standard_task_runner.py:85} INFO - Job 1907: Subtask load_to_gcs_bq.load_account_to_temp
[2025-01-16T03:56:39.625+0000] {task_command.py:416} INFO - Running <TaskInstance: finflow_source_data_pipeline.load_to_gcs_bq.load_account_to_temp manual__2025-01-16T03:55:30.437726+00:00 [running]> on host 39f0a5abbf98
[2025-01-16T03:56:40.514+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='finflow_source_data_pipeline' AIRFLOW_CTX_TASK_ID='load_to_gcs_bq.load_account_to_temp' AIRFLOW_CTX_EXECUTION_DATE='2025-01-16T03:55:30.437726+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-16T03:55:30.437726+00:00'
[2025-01-16T03:56:40.533+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-01-16T03:56:40.551+0000] {connection.py:232} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-01-16T03:56:40.553+0000] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2025-01-16T03:56:40.564+0000] {gcs_to_bigquery.py:376} INFO - Using existing BigQuery table for storing data...
[2025-01-16T03:56:41.444+0000] {gcs_to_bigquery.py:380} INFO - Executing: {'load': {'autodetect': True, 'createDisposition': 'CREATE_IF_NEEDED', 'destinationTable': {'projectId': 'finflow-analytics-production', 'datasetId': 'finflow_data', 'tableId': 'src_accounts_temp'}, 'sourceFormat': 'PARQUET', 'sourceUris': ['gs://finflow-analytics-data/raw/accounts/2025-01-16/accounts.parquet'], 'writeDisposition': 'WRITE_TRUNCATE', 'ignoreUnknownValues': False, 'schema': {'fields': [{'name': 'account_key', 'type': 'INTEGER', 'mode': 'REQUIRED'}, {'name': 'account_id', 'type': 'INTEGER', 'mode': 'REQUIRED'}, {'name': 'customer_key', 'type': 'INTEGER', 'mode': 'REQUIRED'}, {'name': 'product_key', 'type': 'INTEGER', 'mode': 'REQUIRED'}, {'name': 'account_type', 'type': 'STRING', 'mode': 'REQUIRED'}, {'name': 'account_status', 'type': 'STRING', 'mode': 'REQUIRED'}, {'name': 'initial_deposit', 'type': 'FLOAT', 'mode': 'REQUIRED'}, {'name': 'minimum_balance', 'type': 'FLOAT', 'mode': 'REQUIRED'}, {'name': 'balance', 'type': 'FLOAT', 'mode': 'REQUIRED'}, {'name': 'credit_limit', 'type': 'FLOAT', 'mode': 'REQUIRED'}, {'name': 'opened_date', 'type': 'DATE', 'mode': 'REQUIRED'}, {'name': 'closed_date', 'type': 'DATE', 'mode': 'NULLABLE'}, {'name': 'created_at', 'type': 'TIMESTAMP', 'mode': 'REQUIRED'}, {'name': 'updated_at', 'type': 'TIMESTAMP', 'mode': 'REQUIRED'}, {'name': 'ingestion_timestamp', 'type': 'TIMESTAMP', 'mode': 'REQUIRED'}]}}}
[2025-01-16T03:56:41.448+0000] {bigquery.py:1595} INFO - Inserting job ***_finflow_source_data_pipeline_load_to_gcs_bq_load_account_to_temp_2025_01_16T03_55_30_437726_00_00_d3aa38db97622078b755a9d443279ee8
[2025-01-16T03:56:43.100+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/google/cloud/transfers/gcs_to_bigquery.py", line 439, in execute
    job.result(timeout=self.result_timeout, retry=self.result_retry)
  File "/home/airflow/.local/lib/python3.8/site-packages/google/cloud/bigquery/job/base.py", line 922, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/google/api_core/future/polling.py", line 261, in result
    raise self._exception
google.api_core.exceptions.BadRequest: 400 Error while reading data, error message: Invalid timestamp microseconds value 1736999760129543000 of logical type NONE; in column 'created_at' File: gs://finflow-analytics-data/raw/accounts/2025-01-16/accounts.parquet
[2025-01-16T03:56:43.135+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=finflow_source_data_pipeline, task_id=load_to_gcs_bq.load_account_to_temp, execution_date=20250116T035530, start_date=20250116T035638, end_date=20250116T035643
[2025-01-16T03:56:43.212+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 1907 for task load_to_gcs_bq.load_account_to_temp (400 Error while reading data, error message: Invalid timestamp microseconds value 1736999760129543000 of logical type NONE; in column 'created_at' File: gs://finflow-analytics-data/raw/accounts/2025-01-16/accounts.parquet; 1976)
[2025-01-16T03:56:43.331+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-01-16T03:56:43.815+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
